lib-rca/src/main/java/com/facebook/openwifi/librca/models/ClientStats.java

```diff
@@ -0,0 +1,12 @@
+package com.facebook.openwifi.librca.models;
```

Comment:
Are all three of these classes aggregated info/statistics? If so, I think the package name should be `aggregatedstats`?

Reply:
No, inputstats is not aggregated.

Reply:
Maybe we can separate that from the other classes by putting it in a different package called `inputs`? Or maybe they can go here under different sub-packages?